src/app.ts

```diff
@@ -27,6 +27,7 @@ export async function run(args: string[], branch_name: string = "master") {
       expectPort,
     )
     .option("-d, --db-port <port_number>", "dev database port", expectPort)
+    .option("-c, --current-branch", "use current branch")
```

Comment:
I've changed this. We still can't use -r HEAD since we're doing this:
```
function getTemplateUri(branch_name: string): string {
  return `github:originate/create-originate-app/template#${branch_name}`
}
```
But I agree this is a better approach. Thanks!

Reply:
Oh right, sorry! I think `yarn dev tryme -r $(git branch --show-current)` would work.